scripts/ansible

```diff
@@ -185,15 +185,28 @@ $APT_PATH/apt -y install python3-pip
 # https://github.com/iiab/iiab/pull/3022
 pip3 config --global set global.no-cache-dir false
 
-if ! uname -m | grep -q 64; then
+#if ! uname -m | grep -q 64; then
     # echo "2022-11-09: ansible-core 2.12.10+ PPA works on 32-bit RasPiOS, using /etc/apt/sources.list.d/iiab-ansible.list, until upstream wheels -> cryptography is fixed (PR #3421)"
     # $APT_PATH/apt -y --allow-downgrades install ansible-core
-    echo -e "\n\n'pip3 install cryptography==39.0.2' will now run:\n"
-    pip3 install --break-system-packages cryptography==39.0.2 || pip3 install cryptography==39.0.2    # PR #3459 https://www.piwheels.org/project/cryptography/ -- WAS 37.0.4 which as of 2023-01-06 was the "latest compatible with ansible-core available via piwheels.org"
-fi
+#    echo -e "\n\n'pip3 install cryptography==39.0.2' will now run:\n"
+#    pip3 install --break-system-packages cryptography==39.0.2 || pip3 install cryptography==39.0.2    # PR #3459 https://www.piwheels.org/project/cryptography/ -- WAS 37.0.4 which as of 2023-01-06 was the "latest compatible with ansible-core available via piwheels.org"
```

Comment:
Is `pip install cryptography` confirmed to be no longer needed on 32-bit RasPiOS ?

Reply:
The reason the version was pinned at 37.0.4 was because later versions need rust and the builds failed. Remember that screenshot recording the failures https://github.com/iiab/iiab/pull/3459#issuecomment-1378928044? Now that pywheels are now building the latest offerings with rust there is no need to pin the release just use the latest offered.

Reply:
Ok.

(And of course 32-bit RasPiOS smoke testing is also important, to confirm Zero W operations etc.)